src/site/static/benefits-eligibility/eligibility.liquid

```diff
@@ -453,6 +453,7 @@ pageClass: "page-benefits-eligibility"
         va-disability: VA Disability Compensation
         va-pension: Veterans Pension or Survivors Pension
         vta-paratransit: VTA ACCESS Paratransit
+        rtc-clipper: RTC Clipper Card <span class="subtitle">For people with disabilities</span>
```

Comment:
Please change **RTC Clipper Card** to **Regional Transit Connection (RTC) Clipper Card** to alleviate the acronym overload of the public assistance program world.